arbor/threading/threading.hpp

```diff
@@ -114,30 +131,41 @@ class task_group {
     class wrap {
         F f;
         std::atomic<std::size_t>& counter;
+        exception_struct& exception_status;
```

Comment:
I just noticed that the private members of this class don't have `_` on the end of the variable names, e.g. `counter` should be `counter_`, `f` should be `f_`, etc.